tubesize/src/popup.tsx

```
@@ -0,0 +1,128 @@
+    }, []);
+
+    return (
+        <>
+            <div className="header">
+                <div className="title" title={videoData?.data?.title}>
+                    {videoData?.data?.title ?? "TubeSize"}
+                </div>
+                <span className="duration" id="duration-display">
+                    {videoData?.data?.duration}
+                </span>
+                <button id="optionsBtn" onClick={optionsBtnOnChange()}>
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)